docs/commands/super.md

```diff
@@ -659,20 +659,32 @@ the `super` command, but it turns out that `super` can hold its own when
 compared to other analytics systems.
 
 To illustrate comparative performance, we'll present some informal performance
-measurements among `super`,
-[`DuckDB`](https://duckdb.org/),
-[`ClickHouse`](https://clickhouse.com/), and
-[`DataFusion`](https://datafusion.apache.org/).
+measurements among SuperDB,
+[DuckDB](https://duckdb.org/),
+[ClickHouse](https://clickhouse.com/), and
+[DataFusion](https://datafusion.apache.org/).
 
 We'll use the Parquet format to compare apples to apples
 and also report results for the custom columnar database format of DuckDB
-and the Super Binary format used by `super`.
+and the [Super Binary](../formats/bsup.md) format used by `super`.
 We tried loading our test data into a ClickHouse table using its
 [new experimental JSON type](https://clickhouse.com/blog/a-new-powerful-json-data-type-for-clickhouse)
 but those attempts failed with "too many open files".
 
-As of this writing in November 2024, we're using the latest version 1.1.3 of `duckdb`.
-version 24.11.1.1393 of `clickhouse`, and v43.0.0 of `datafusion-cli`.
+The detailed steps shown [below](#appendix-2-running-the-tests) can be reproduced via
+[automated scripts](https://github.com/brimdata/super/blob/main/scripts/super-cmd-perf).
```

Comment:
Since the scripts currently only exist on this branch, this hyperlink will fail until this PR is merged.